msg/ChargingStatus.msg

```diff
@@ -0,0 +1,13 @@
+# This message defines the battery charging process status 
+
+# Charger types
+uint8 UNKNOWN = 0
+uint8 WIRED = 1
+uint8 WIRELESS = 2
+
+std_msgs/Header header
+bool charging               # True if battery is being charged
+float32 current             # Power supply total current (A)
+float32 current_battery_1   # Power supply current (A) for battery 1
+float32 current_battery_2   # Power supply current (A) for battery 2 (NaN for single battery configuration)
```

Comment:
What about using array `batteries_current`. Array is better scalable and not NaN would be unused.

Reply:
Right, array solves the scalability issue and this was my initial implementation. Unfortunately, it comes with the drawback of not strictly binding values in an array with battery numbers.